ws/nats/publisher.go

```diff
@@ -25,7 +27,10 @@ const (
 	maxFailureRatio = 0.6
 )
 
-var _ ws.Service = (*natsPubSub)(nil)
+var (
+	_               ws.Service = (*natsPubSub)(nil)
+	errInvalidTopic            = errors.New("invalid topic")
+)
```

Comment:
We separate error vars from interface checking ones.

Reply:
Sorry @anovakovic01 i don't understand, for  example ws/api/transport declare its own errors, which is the best place to define it?

Reply:
Sorry, I wasn't clear enough. What I meant by this is that you should separate vars into logical blocks. What I mean by this is that you first declar your errors and then this var that checks interface implementation:
```
var (
    err1 = ...
    err2 = ...
)

.
.
.

var _ ws.Service = (*natsPubSub)(nil)
```

Reply:
@beres is this one done?

Reply:
Yes with the commit 638efec